pkg/diagnose/runner.go

```diff
@@ -48,7 +48,23 @@ type diagSuiteFilter struct {
 }
 
 // Output summary
-func (c *counters) summary(w io.Writer) {
+func (c *counters) summary(w io.Writer, toJSON bool) {
+	if toJSON {
+		counts := map[string]int{
+			"total":   c.total,
+			"success": c.success,
+			"fail":    c.fail,
+			"warning": c.warnings,
+			"error":   c.unexpectedErr,
+		}
+		jsonResponse, err := json.MarshalIndent(counts, "", "  ")
+		if err != nil {
+			return
```

Comment:
shoudn't we log the error here to allow debugging through logs when (for some reason) this fails?

Reply:
it is done thx